core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.ts

```diff
@@ -81,6 +84,25 @@ angular.module('oppia').controller('TranslationModalController', [
       });
 
     $scope.onContentClick = function($event) {
+      // All the child elements are copied into paragraphChildrenElements
+      // if the copied snippet is a paragraph.
+      // Checks whether there are any mathematical equations are copied
+      // as mathematical equations are also wrapped by <p> elements.
+      // If there are no mathematical equations copied, shows the paragraph
+      // copy error. It is related to issue 11683.
```

Comment:
Prefer to discuss the "why" rather than the "how" in code comments, e.g. you don't need to explain the conditional logic in the comment, just why not having a math element is considered a paragraph copy error.

It might be clearer if instead of `hasParagraphCopyError` it was something like `triedToCopyText`. And then we explain that copied text would exist in a paragraph element.

Reply:
Done